realm/realm-library/src/main/java/io/realm/HandlerController.java

```diff
@@ -786,4 +786,23 @@ public boolean isAutoRefreshEnabled() {
         return autoRefresh;
     }
 
+    /**
+     * When a Class is removed while {@link RealmResults} derived from the class exist, it is required
```

Comment:
When the schema defintion for a Class is removed while a RealmResults is still present, the underlying TableView should be replaced by an always empty tableview instead.